infra/recipes/docker-compose/common/hadoop/hadoop.env

```diff
@@ -43,3 +43,6 @@ MAPRED_CONF_yarn_app_mapreduce_am_env=HADOOP_MAPRED_HOME=/opt/hadoop-3.2.1/
 MAPRED_CONF_mapreduce_map_env=HADOOP_MAPRED_HOME=/opt/hadoop-3.2.1/
 MAPRED_CONF_mapreduce_reduce_env=HADOOP_MAPRED_HOME=/opt/hadoop-3.2.1/
 
+AZURE_CLIENT_ID=<client-id>
```

Comment:
So these Azure specific details need to be manually populated, right?

Reply:
Yes, correct

Reply:
why are these needed?

Reply:
They are not actually. Good catch